hub_adapter/routers/kong.py

```
@@ -321,18 +322,18 @@ async def list_specific_project(
-        data_store_id: Annotated[uuid.UUID | str, Body(description="UUID of the data store or 'service'")],
-        project_id: Annotated[uuid.UUID | str, Body(description="UUID of the project")],
-        methods: Annotated[list[HttpMethodCode], Body(description="List of acceptable HTTP methods")] = ["GET"],
-        protocols: Annotated[
-            list[ProtocolCode],
-            Body(description="List of acceptable transfer protocols. A combo of 'http', 'grpc', 'grpcs', 'tls', 'tcp'"),
-        ] = ["http"],
-        ds_type: Annotated[DataStoreType, Body(description="Data store type. Either 's3' or 'fhir'")] = "fhir",
+    settings: Annotated[Settings, Depends(get_settings)],
+    data_store_id: Annotated[uuid.UUID | str, Body(description="UUID of the data store or 'service'")],
+    project_id: Annotated[uuid.UUID | str, Body(description="UUID of the project")],
+    methods: Annotated[list[HttpMethodCode], Body(description="List of acceptable HTTP methods")] = ["GET"],
```

Ruff (0.15.2)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)